Representable/representable
# Recaptcha works on backend! front end error also shows up when forget to click on it

to test:
need to add updated private key in environment - check bitwarden or dm me
go to the entry page and dont click on recaptcha, check for error next to it

checking if it prevents entries from being submitted can be done using locust and I can attach a screenshot later today
Also need to fix the box when it gives an error like: 'verification expired' 
check screenshot below:
![image](https://user-images.githubusercontent.com/15281033/103423449-9c917500-4b74-11eb-97cd-e48d311a957d.png)
